### PR TITLE
OHRM5X-820: Module under development page added

### PR DESCRIPTION
**Pull request by VageeshOrangeHRM**
This PR Contains following changes:
1. Module Under development page added.
2. Module Under development PNG Image added.
3. OHRM5X-767: Back End added.
4. New front end controller added  for module under development.
5. New Subscriber added for module under development.
6. Screen module disabling dbscript lines removed. 
7. Performance module default page link added with screen permissions.

### CODE REVIEW

File: symfony/plugins/orangehrmCorePlugin/Subscriber/ModuleUnderDevelopmentSubscriber.php [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    {
+        $moduleUnderDevelopment = ['time', 'recruitment', 'performance', 'directory', 'dashboard', 'maintenance', 'buzz'];
+        foreach ($moduleUnderDevelopment as $module){
+            if ($event->isMasterRequest()) {
+                if ($this->getTextHelper()->strStartsWith($event->getRequest()->getPathInfo(), '/'.$module)) {
+                    throw new RequestForwardableException(ModuleUnderDevelopmentController::class . '::handle');
+                }
+            }
+        }
+    }
```
[the remark this answers is not in the document]
Comment (VageeshOrangeHRM) [reply symfony/plugins/orangehrmCorePlugin/Subscriber/ModuleUnderDevelopmentSubscriber.php]: done

File: symfony/plugins/orangehrmCorePlugin/Subscriber/ModuleUnderDevelopmentSubscriber.php [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+        return [
+            KernelEvents::REQUEST => [
+                ['onRequestEvent', 100],
+            ],
+        ];
+    }
+
+    public function onRequestEvent(RequestEvent $event)
+    {
+        $moduleUnderDevelopment = ['time', 'recruitment', 'performance', 'directory', 'dashboard', 'maintenance', 'buzz'];
```
[the remark this answers is not in the document]
Comment (VageeshOrangeHRM) [reply symfony/plugins/orangehrmCorePlugin/Subscriber/ModuleUnderDevelopmentSubscriber.php]: done

**Review by Super-Chama — APPROVED** [review]

File: symfony/plugins/orangehrmCorePlugin/Subscriber/ModuleUnderDevelopmentSubscriber.php [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+            KernelEvents::REQUEST => [
+                ['onRequestEvent', 100],
+            ],
+        ];
+    }
+
+    public function onRequestEvent(RequestEvent $event)
+    {
+        if ($event->isMasterRequest()) {
+            $modules = self::MODULE_UNDER_DEVELOPMENT; // [indirectly reference the const] looping directly may case run time exception
```
Comment (RajithaKumara) [inline symfony/plugins/orangehrmCorePlugin/Subscriber/ModuleUnderDevelopmentSubscriber.php:58]: are you sure about the comment?

File: symfony/plugins/orangehrmCorePlugin/Subscriber/ModuleUnderDevelopmentSubscriber.php [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+            foreach ($modules as $module) {
+                if ($this->getTextHelper()->strStartsWith($event->getRequest()->getPathInfo(), '/' . $module)) {
+                    throw new RequestForwardableException(ModuleUnderDevelopmentController::class . '::handle');
+                }
+            }
```
Comment (RajithaKumara) [inline symfony/plugins/orangehrmCorePlugin/Subscriber/ModuleUnderDevelopmentSubscriber.php:63]: can you try
```php
            foreach (self::MODULE_UNDER_DEVELOPMENT as $module) {
                if ($this->getTextHelper()->strStartsWith($event->getRequest()->getPathInfo(), '/' . $module)) {
                    throw new RequestForwardableException(ModuleUnderDevelopmentController::class . '::handle');
                }
            }
```

**Review by RajithaKumara — APPROVED** [review]

File: symfony/plugins/orangehrmCorePlugin/Subscriber/ModuleUnderDevelopmentSubscriber.php (hunk 4 above)
Comment (VageeshOrangeHRM) [reply symfony/plugins/orangehrmCorePlugin/Subscriber/ModuleUnderDevelopmentSubscriber.php]: done

File: symfony/plugins/orangehrmCorePlugin/Subscriber/ModuleUnderDevelopmentSubscriber.php (hunk 3 above)
Comment (VageeshOrangeHRM) [reply symfony/plugins/orangehrmCorePlugin/Subscriber/ModuleUnderDevelopmentSubscriber.php]: not sure, one documentation said like that, hence i followed that way

**Pull request merged**